cli/pkg/commands/images/images.go

```diff
@@ -206,7 +234,44 @@ func (m imagesModel) viewListModelImages() string {
 	return common.TableBaseStyle.Render(t.View()) + "\n"
 }
 
-// Cmd represents the images command
+func (m imagesModel) viewDescribeModelImages() string {
+
+	var parameters []string
+	var paramItem string
+	for key, value := range m.selectedImage.BuildParams {
+		if key == "target-image-tag-references" {
```

Comment:
```suggestion
		if ok, keySlice := key.([]string); ok {
```
I think it's better if we check the type of the key instead of the key name. 

Inside this if, `keySlice` will be a slice of strings and you can use `for...range` to iterate over it.

Reply:
yep this looks better approach, have changed it now